tests/test_history.py

```diff
@@ -114,47 +114,51 @@ def run_show_cmd(hist_args, commands, base_idx=0, step=1):
+            hist.append({'inp': cmd, 'rtn': 0, 'ts':(ts+1, ts+1.5)})
 
-        # Verify an implicit "show" emits the entire history.
-        for x in run_show_cmd([], cmds):
+        # Verify an implicit "session" emits session history
+        for x in run_session_cmd([], cmds):
             yield x
 
-        # Verify an explicit "show" with no qualifiers emits the entire history.
-        for x in run_show_cmd(['show'], cmds):
+        # Verify an explicit "session" with no qualifiers emits
+        # session history.
+        for x in run_session_cmd(['session'], cmds):
             yield x
 
-        # Verify an explicit "show" with a reversed qualifier emits the entire
-        # history in reverse order.
-        for x in run_show_cmd(['show', '-r'], list(reversed(cmds)),
-                              len(cmds) - 1, -1):
+        # Verify an explicit "session" with a reversed qualifier
+        # emits session history in reverse order.
+        for x in run_session_cmd(['session', '-r'], list(reversed(cmds)),
+                                 len(cmds) - 1, -1):
             yield x
 
-        # Verify that showing a specific history entry relative to the start of the
-        # history works.
-        for x in run_show_cmd(['show', '0'], [cmds[0]], 0):
+        # Verify that showing a specific history entry relative to
+        # the start of the history works.
+        for x in run_session_cmd(['session', '0'], [cmds[0]], 0):
             yield x
-        for x in run_show_cmd(['show', '1'], [cmds[1]], 1):
+        for x in run_session_cmd(['session', '1'], [cmds[1]], 1):
             yield x
 
-        # Verify that showing a specific history entry relative to the end of the
-        # history works.
-        for x in run_show_cmd(['show', '-2'], [cmds[-2]], len(cmds) - 2):
+        # Verify that showing a specific history entry relative to
+        # the end of the history works.
+        for x in run_session_cmd(['session', '-2'], [cmds[-2]],
+                                 len(cmds) - 2):
             yield x
 
         # Verify that showing a history range relative to the start of the
         # history works.
-        for x in run_show_cmd(['show', '0:2'], cmds[0:2], 0):
+        for x in run_session_cmd(['session', '0:2'], cmds[0:2], 0):
             yield x
-        for x in run_show_cmd(['show', '1::2'], cmds[1::2], 1, 2):
+        for x in run_session_cmd(['session', '1::2'], cmds[1::2], 1, 2):
             yield x
 
         # Verify that showing a history range relative to the end of the
         # history works.
-        for x in run_show_cmd(['show', '-2:'], cmds[-2:], len(cmds) - 2):
+        for x in run_session_cmd(['session', '-2:'], 
+                                 cmds[-2:], len(cmds) - 2):
             yield x
-        for x in run_show_cmd(['show', '-4:-2'], cmds[-4:-2], len(cmds) - 4):
+        for x in run_session_cmd(['session', '-4:-2'], 
+                                 cmds[-4:-2], len(cmds) - 4):
```

Comment:
All of these show tests should be restored and potenially new session tests added.